ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -345,6 +346,20 @@ void testCreateWithAadTokenProviderAuthentication() {
         assertTrue(canAuthenticate(engineCsb));
     }
 
+    @Test
+    void testCloudInfoWithCluster() throws DataServiceException {
```

Comment:
Yep.
In theory the content of both default and retrieved settings is the same so you can compare it as well